ui/tests/acceptance/clients/counts/overview-test.js

```diff
@@ -318,6 +318,9 @@ module('Acceptance | clients | overview | sync not in license', function (hooks)
 
   hooks.beforeEach(async function () {
     this.store = this.owner.lookup('service:store');
+    // mocks endpoint for no additional license modules or CE edition
```

Comment:
technically the `sys/license/features` endpoint fails for CE but that's neither here nor there

Reply:
oh that's good to know! someone told me it was an empty array.

Reply:
The features service holds an empty array, but afaik we don't even call the license/features endpoint if it's CE so it stays an array